scm/src/GFS_typedefs.F90

```diff
@@ -4104,8 +4110,13 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
        'max_lon=',max_lon,' max_lat=',max_lat,' min_lon=',min_lon,' min_lat=',min_lat,       &
        ' rhc_max=',Model%rhcmax
 
-!--- set nrcm
-    Model%nrcm = 2
+!--- set nrcm 
+
+    if (Model%ras) then
```

Comment:
This is not new, but I assume it was forgotten when the RAS updates were brought into SCM in the first place.

Reply:
Yes.